tests/models/test_models.py

```diff
@@ -471,17 +479,22 @@ def test_new_setup_compat():
     model = SCVI(adata)
     model.view_anndata_setup(hide_state_registries=True)
 
+    registry = adata_manager.registry[_constants._FIELD_REGISTRIES_KEY]
```

Comment:
what is going on here?

Reply:
oops poor naming. But basically keeping this test working even if new fields are added to SCVI (e.g. size factor) - not supported by backwards compatibility.